config/config.yaml

```diff
@@ -9,10 +9,9 @@ kraken:
     chaos_scenarios:
         # List of policies/chaos scenarios to load
         - hog_scenarios:
-            - scenarios/kube/cpu-hog/input.yaml
-            - scenarios/kube/memory-hog/input.yaml
-            - scenarios/kube/io-hog/input.yaml
-            - scenarios/kube/io-hog/input.yaml
+            - scenarios/kube/cpu-hog.yml
```

Comment:
Also need to update/replace hog scenarios documentation with the new example configs and parameters: https://github.com/krkn-chaos/krkn/blob/main/docs/arcaflow_scenarios.md.